### DISCUSSION TITLE
⚪️ YouTube Reborn v5 Development Discussion

### DISCUSSION THREAD

**Discussion by arichornlover** [arichornlover/uYouEnhanced#256] [General]
Hello! I’m currently focusing on YouTube Reborn v5 development. And this announcement has been made because YouTube Reborn v5.0.0 is a big change with the changes below
- New YouTube Reborn Menu (similar to Ginsudev/YTMusicUltimate’s Menu)
- Localization Support (a long waited feature for YouTube Reborn)
- More Options & Enhancements

It is also painful to make it all work as intended and the New YouTube Reborn Menu is mostly gonna be with YTMusicUltimate’s Menu which is good to make them consistent I guess?

YouTube Reborn v5.0.0
 https://github.com/arichornlover/YouTube-Reborn-v5/tree/main-nightly

**Comment by arichornlover** [comment]
https://github.com/arichornlover/YouTube-Reborn-v5/releases/tag/v5.0.0-beta
First Beta of YouTube Reborn v5 released!
tell me what you think of the localization support and the New Interface.
if the new interface doesn’t feature rounded corners then it means that the new Interface in YouTube Reborn did not work.

**Reply by ego-lay-atman-bay** [reply]
Can you share some screenshots?

**Reply by arichornlover** [reply]
![IMG_1431](https://github.com/arichornlover/uYouEnhanced/assets/78001398/19e0ea69-5cbe-48a6-9545-995aedb4540a)
This is what the YouTube Reborn v4 Menu looks like if using v4.2.3 (not accurate, it’s an old screenshot from Google but I do believe it was from YouTube Reborn v4)

but right now I’m gonna test YouTube Reborn v5 and see if the new menu works and then I show the new screenshots?

**Reply by ego-lay-atman-bay** [reply]
Thanks. I just want to get a preview of it without actually installing it. I might give it a try because it looks good, and I'm assuming v5 would look better.

**Reply by arichornlover** [reply]
https://github.com/arichornlover/uYouEnhanced/assets/78001398/f04ac266-3245-4b8e-bf2d-9631d19a7ae6

heres what YouTube Reborn v5-beta1 looks like. As you can see Localization isn’t working at the moment but the new menu is working.

**Reply by thaibm92** [reply]
I tried installing it, but I don't see the MENU option, where is it?

**Reply by arichornlover** [reply]
It should be on the navigation bar on the app. And I think the logo might be invisible which is another problem with the localization, but it can be pressed.
and keep in mind it’s a Beta Release, a lot of stuff isn’t working as intended.

**Reply by thaibm92** [reply]
Indeed, the logo doesn't appear, if you click on it, the menu will appear. v5 is very good, please consider adding the premium logo to YouTube.

**Reply by arichornlover** [reply]
although v5 is great. there is a problem with the feature “Reorder Tabs”. It crashes when tapping and holding it and the labels of it is invisible aswell 😅
I gotta figure out how to resolve it.
and the Premium Logo is gonna be implemented to YouTube Reborn

**Reply by arichornlover** [reply]
I just recently released Beta 2 of YouTube Reborn v5, tell me what you think of the new credits menu in YouTube Reborn Menu ;)

if it even works

**Reply by thaibm92** [reply]
https://github.com/arichornlover/uYouEnhanced/assets/68208369/8c75e202-283f-4ca3-accf-e991cb4759bb

The structure looks nice but I don't see any content for both light and dark mode. I sign with the developer's certificate.

**Reply by arichornlover** [reply]
I might not include the ipas in the YouTube Reborn tweak because of the text problem you are having., I’ll build the YouTubeRebornPlus version and then try it out, give me a moment

**Reply by arichornlover** [reply]
Sorry for the long wait, here you go!
https://github.com/arichornlover/YouTubeRebornPlus/releases/tag/v18.49.3-5.0.0-B2.1
the explanation on why the ipa wasn’t showing any texts or icons earlier is because the ipa you used did not build with the localization but however, if I inject the stuff onto YouTubeRebornPlus the localization gets implemented so that’s weird 😭
anyways, I just hope this version works!

**Reply by thaibm92** [reply]
https://github.com/arichornlover/uYouEnhanced/assets/68208369/6571cee0-738c-4807-82dd-d53d99436e95

Maybe it works fine, I checked a few things, as you said REother tabs are still not displayed. It's great, I'm waiting for you to add the premium logo.

**Reply by arichornlover** [reply]
The premium logo, that is something I’m planning to add soon, and the Reorder Tabs functionality is broken which explains the invisible text labels and if you didn’t know you can also add your localization to the https://github.com/arichornlover/youtube-reborn-v5 if you want, I just added new strings for the new stuff I’m adding.

**Comment by arichornlover** [comment]
Y’know what, I’m gonna implement some of the YouTube Reborn Features into Theme Options for uYouEnhanced
LowContrastMode & Dark Theme will have a color picker which upgrades and gets more personalization on YouTube 🎆

**Comment by arichornlover** [comment]
Hey! I recently released a new Beta!
 YouTube Reborn v5.0.0 Beta 5
Included the following changes below!

a working `Reorder Tabs` in Settings
(It isn’t fully working, it changes nothing to the app right now)
![IMG_2584](https://github.com/arichornlover/uYouEnhanced/assets/78001398/c89b7346-9ebe-4c66-b978-daae089f362d)

an improved `View Downloads` Menu!

the menu is definitely similar to the old one but however, in Beta 5, it includes a search bar and icons on the Tab below! it is an Improvement!
![IMG_2587](https://github.com/arichornlover/uYouEnhanced/assets/78001398/707ea9b5-0a7b-4be8-9047-55ff3f3554ad)

OP Button can be added under the Video Player!

another way to download videos using the button under the video player!
(however, that button can’t be shown on Beta 5, it needs a few adjustments before I make it appear.)
![IMG_2590](https://github.com/arichornlover/uYouEnhanced/assets/78001398/46108cfc-ad51-4d61-8d2b-961b2595860b)


other changes
- fixed `Credits` Menu Images not loading when calling the menu once

**Reply by thaibm92** [reply]
https://github.com/arichornlover/uYouEnhanced/assets/68208369/aa032b63-d7e3-4526-9607-31d26f31cae5

![image](https://github.com/arichornlover/uYouEnhanced/assets/68208369/1d69e84a-dfa4-46df-8842-b815e819ca3d)

![image](https://github.com/arichornlover/uYouEnhanced/assets/68208369/a5ee66da-3ddf-4e4f-afb9-ec6ce8f5d51f)


This is all I see, reorder, speed have error, Is that an error?

**Reply by arichornlover** [reply]
1. Reorder Tabs is still in WIP, however, in order to use it you have to tap and hold on one of them, I might change it but not sure.
2. as for the speed error, why did it disappear 😭
I’ll check out the speed issue.

**Reply by arichornlover** [reply]
Hey @Thaibm92
what do you think of [v5.0.0 Beta 6](https://github.com/arichornlover/YouTubeRebornPlus/releases/tag/v19.03.2-5.0.0-B6)
this beta updates the `View Downloads` Menu once again and improved the Search Bar Functionality because it wasn’t good enough. And there were other changes in the Beta in the Changelog if you want the rest.

**Reply by thaibm92** [reply]
That's so great!
This update View Downloads is beautiful, but the "ALL" section is not working yet, and if can display a thumbnail image and display a name then perfect
Additionally, I see these functions still not working:
1. Reoder tab Not working correctly yet
2. Can't select "App Version Spoofer"

https://github.com/arichornlover/uYouEnhanced/assets/68208369/e58003fb-4fd7-42bd-b231-1155360cc2f0

**Reply by arichornlover** [reply]
So far so good.
it also does suck that Reorder Tabs is still not working and the App Version Spoofer Text Field not appearing. I’m trying to find smarter ways for those implementations.

and as for the `All` tab, I didn’t test it yet and since you did, I can easily do some fixes for it and try to make it look better.

**Reply by arichornlover** [reply]
Beta 6.1 Update: I fixed the `All` Tab and I fixed the App Version Spoofer text field that is now a visible text field, but still need to modify that since it is positioned in the version spoofer option in YouTube Reborn Settings. I’m trying my best with more improvements! 😅

**Reply by thaibm92** [reply]
The ALL tab works now, but the display name is terrible and it makes the search function useless. Can you improve it to have the image and name exactly like the original video?
The fake version of the app may not work properly, currently, I see it showing failed.
![z5115403850143_e615cdd1094b51278a83eb125010f60d](https://github.com/arichornlover/uYouEnhanced/assets/68208369/aacf7739-83e5-42a8-925f-c490601f9286)
![z5115397769388_62a439dd83a9e799539fcceba90cf78c](https://github.com/arichornlover/uYouEnhanced/assets/68208369/161485c5-b9a3-43be-84a4-3ab53ef5a88c)

**Reply by arichornlover** [reply]
Where the heck did the Text Field in the version spoofer go? It was present in Beta 6.1?

edit: I see it. It’s inside the Version Spoofer Option, whoops.. (this issue only happens in Beta 6.1 & 7 I think.)

**Reply by thaibm92** [reply]
when I turn it on, it's hidden below, now, is 6.1

**Comment by arichornlover** [comment]
heres what YouTube Reborn v5 looks like! as of February 2024! Do you think it’s better then YouTube Reborn v4?

![IMG_2605](https://github.com/arichornlover/uYouEnhanced/assets/78001398/17246341-d885-4bd6-a9f3-5f3869a9cb60)
![IMG_2606](https://github.com/arichornlover/uYouEnhanced/assets/78001398/ae7ce29b-c6ac-4a12-b6e0-edec102171a1)
![IMG_2607](https://github.com/arichornlover/uYouEnhanced/assets/78001398/513c4587-3797-4b63-ba9c-ca9ecb52f25b)

**Reply by arichornlover** [reply]
Note: Color Options in the Menu is subject to change. I’m not satisfied with how I wasted like 3 Rows space for the Color Options in the YouTube Reborn Menu.🙁


also. I’m sorry for removing the Reorder Tabs section. I failed to do the implementation for that and I’m sorry.

**Reply by thaibm92** [reply]
Great snow, except for the way the names of downloaded video files are displayed, it’s very ugly ^^! 

Can you improve it? and if it is possible to use the functions of the rebornPlus version in the same Teawk, then verry good!

**Reply by arichornlover** [reply]
YTMusicUltimate has a better implementation with the Titles when downloading a video so I could go with that version.
at this point YouTube Reborn is slowly starting to become YTMusicUltimate (and since it is, I did update Dayanch’s Credit description in the Credits Section)

**Comment by arichornlover** [comment]
> heres what YouTube Reborn v5 looks like! as of February 2024! Do you think it’s better then YouTube Reborn v4?
> 
> ![IMG_2605](https://private-user-images.githubusercontent.com/78001398/302501160-17246341-d885-4bd6-a9f3-5f3869a9cb60.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc5NTI2NzgsIm5iZiI6MTcwNzk1MjM3OCwicGF0aCI6Ii83ODAwMTM5OC8zMDI1MDExNjAtMTcyNDYzNDEtZDg4NS00YmQ2LWE5ZjMtNWYzODY5YTljYjYwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjE0VDIzMTI1OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY2YjMwZWFlMGEyN2E3ZWMxZmM4YTg0ZGM1YjcxYmZiZTU1ZjM3MTkyMTJlYzhhNTdkMTMwYzFkZGQ2ZDlmM2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.wuj4kNqvi88UTt4Vqn4mlk5Bh4wfVj17duI9DfbJbbI)

In Beta 10, the menu is simplified especially `Color Options` getting the same treatment like `View Downloads`
![IMG_2618](https://github.com/arichornlover/uYouEnhanced/assets/78001398/e7800b13-8dce-47ca-a485-00d6cf47fcca)
![IMG_2617](https://github.com/arichornlover/uYouEnhanced/assets/78001398/cc53644a-0e93-4047-97a5-3030641eee86)

and as for the request thaim92’s for the `View Downloads` Improvement, it’s sadly not working yet. However I am removing blue accessibility button’s shown because it takes space. And I will somehow give the spaces titles and give them thumbnails next.

**Comment by arichornlover** [comment]
https://github.com/arichornlover/uYouEnhanced/assets/78001398/3724a737-3b13-4ab6-baa5-b0fb235c1c3e

Here’s a showcase of the new changes I’ve done.
- Replaced accessory button on the right with swipe-to-left gesture button. (I’m referring to the new one with a blue background that shows the 3 dots icon. easy explanation I guess.)

- Added the ability to modify the file name using `Edit File Name`. - this was also something that was easy to implement and this idea wouldn’t have happened if it weren’t to @thaibm92’s suggestion up top! and I’ll still find a way to fix the titles and make the thumbnails appear if it’s possible! 😄

- Added Background Playback Support to `View Downloads`! - one of the most important features gets added to YouTube Reborn!

**Reply by thaibm92** [reply]
great i am waiting for the next version and good news from you

**Reply by thaibm92** [reply]
Hi Arichornlover, 
The features from beta10 seem to no longer exist, are you planning something else, or did I miss your information?

![image](https://github.com/arichornlover/uYouEnhanced/assets/68208369/469f03f3-825b-4537-81dc-e1305270eede)
![image](https://github.com/arichornlover/uYouEnhanced/assets/68208369/225db687-2e65-4faa-9b88-111d9d85c0b7)

**Reply by arichornlover** [reply]
It’s experimental, its only shown in the audio tab because there’s some issues I gotta fix before putting it in the Video & All Tabs.

**Reply by thaibm92** [reply]
Oh thanks, I saw it

**Reply by thaibm92** [reply]
I just checked and got an error here, I think it's serious!
1. Hidden by default when opened (large space)
![image](https://github.com/arichornlover/uYouEnhanced/assets/68208369/85cdf79d-1e81-4ee5-8cd6-ecb38263b648)

2. OP button doesn't work and causes the app to quit (if you click the OP letter in the overlay before clicking the button it will work)
![image](https://github.com/arichornlover/uYouEnhanced/assets/68208369/7697a725-87d5-45b3-80f7-8f239a3e6e4f)

**Reply by arichornlover** [reply]
1. Large space, I’m planning replacing the functionality with the one from uYouEnhanced and it will no longer have those gaps soon.

2. OP Button does work. It only works if the video is paused. It’s supposed to automatically pause the video if it was pressed but couldn’t figure that one out yet.

**Reply by thaibm92** [reply]
I just discovered an aesthetic error missing the ICON of the systemBlue color option

**Reply by arichornlover** [reply]
I might’ve typed in the wrong icon.. I gotta find a actual icon for it.

**Reply by thaibm92** [reply]
The current icon doesn't have the exact meaning and neither does the Shorts ^^, if you can, please change it to make it better, and it seems like the options in the video player aren't working, I turned it on but didn't see any change

**Reply by arichornlover** [reply]
I changed some of the icons. Do you think they are better?

**Reply by arichornlover** [reply]
https://github.com/arichornlover/uYouEnhanced/assets/78001398/b8b665ee-7551-4302-8e8c-1fc5fca56847
So uhhh, I fixed the “Import to Downloads” button. it definitely works, but however, it does the opposite with how I wanted the feature “Import to Downloads” to be. This was supposed to let you import any of your downloaded files from the View Downloads menu to the downloads folder but somehow I made it work the opposite if you have seen the video already. 😭

Do y’all want me to remove this accidental feature or should I just keep it in?

**Reply by thaibm92** [reply]
Could you consider icons like the below:
1. Video options
![image](https://github.com/arichornlover/uYouEnhanced/assets/68208369/1353b1dd-013b-4d52-ae1d-6f669809deaf)
![image](https://github.com/arichornlover/uYouEnhanced/assets/68208369/57e5466a-0e81-400d-826d-4fbbfda13d11)
3. Video player options
![image](https://github.com/arichornlover/uYouEnhanced/assets/68208369/ef90d9dc-d3cd-445a-8dd9-fbaa50c80bd3)
![image](https://github.com/arichornlover/uYouEnhanced/assets/68208369/604539d3-40e0-4f03-ace7-4d5eee9c7ff5)
4. Shorts
![image](https://github.com/arichornlover/uYouEnhanced/assets/68208369/98fb6bb1-a9d4-4d80-86dc-1793a8426478)
![image](https://github.com/arichornlover/uYouEnhanced/assets/68208369/63747e40-ce8e-406f-ab81-7d5d20508a1a)
![image](https://github.com/arichornlover/uYouEnhanced/assets/68208369/efe6e07b-35f0-4545-b0ed-60d1549059d0)

**Reply by thaibm92** [reply]
“Import to download” is a good feature that should be kept, and find the reason why it is not working correctly

**Reply by arichornlover** [reply]
I’m making a new action called “Export File” since the other one can be useful if you think about it.

**Reply by arichornlover** [reply]
![IMG_2657](https://github.com/arichornlover/uYouEnhanced/assets/78001398/c5b1bc32-94d7-42c6-8f1b-9e3f0b76df33)
Improved the UI in the Audio Tab on Beta 11! I have placed the file formats under the title of the file. Do you think this looks better?

**Comment by arichornlover** [comment]
Announcement! Not related to the tweak above but this one is about uYouEnhanced! I ported the Color Customization and Clear Cache Button onto uYouEnhanced successfully! Hope that’s useful!

**Reply by arichornlover** [reply]
# for uYouEnhanced Users Only (this doesn’t effect YouTube Reborn.)

I did forget to mention that you may have to enable certain things to active the color customization.

**Theme Customization:** in order to make the app change the theme color if used. You have tap Theme Options which can only be found in uYouEnhanced Settings. Once you tapped it you should see the selection menu. Now select `Custom Dark Theme` and it should activate the custom color you had setted up.

**LowContrastMode/Tint Customization:** same as Theme Customization. Go to uYouEnhanced Settings, once you do go to UI Interface Options and you should see an option called `Low Contrast Mode`, enable it and then once enabled press the `Low Contrast Mode Selector` and select `Custom Color` to activate the custom color aswell.

I’m sorry that this is confusing but I didn’t want this enabled by Default in uYouEnhanced since it’s possible to encounter issues if this was used by default.

**Reply by thaibm92** [reply]
I can't build it, pls check help me

**Reply by arichornlover** [reply]
https://github.com/arichornlover/YouTube-Reborn-v5/actions/runs/8243772262/artifacts/1317360978
use this one for now. It’s Reborn v5 Beta 11

you are having errors because the new Import Button I was making wasn’t finished in YouTube Reborn.

**Reply by arichornlover** [reply]
Here’s YouTube Reborn v5 Beta 13! https://www.dropbox.com/scl/fi/nhq1d4vuogc405rb0newe/YouTube.Reborn.v5.0.0.deb?rlkey=rjaghujuyik1can8m45p0tui7&dl=1

changes I’ve done:
- improved icons in the ui. Requested by @thaibm92
- added an App Compatibility Checker **<-- subject to change**
   - this will check if the app version is v19.06.2 or higher then the tweak will work as normal
   - but, if the app checks that the app version is lower then v19.06.2 then it will show a message and close the Reborn menu. and the reason was due to the updated `Downloads`
- MACROS Support (basically the source-code is now simplified and easier)

and I apologize for it still being in the Beta phase. I still have a feeling of it not being done.

**Reply by thaibm92** [reply]
I just checked and found some serious errors
1. Once downloaded, the interface does not automatically disappear
2. The application will exit when pulled to the left in the toolbar ( download, like, share ....)
Note: YouTube IPA 19.12.4 or 19.13.1
https://github.com/arichornlover/uYouEnhanced/assets/68208369/8767b5aa-a184-4e0e-b60e-663751690244

**Reply by arichornlover** [reply]
Ok. This one is my bad, thanks for letting me know. I’ll change it in a bit.

**Reply by thaibm92** [reply]
On YouTubeRepornPlus, I also had the problem of not being able to open the video, the application automatically exits when clicking on the video with IPA version 19.13.1

**Reply by arichornlover** [reply]
Turn off iSponsorBlock?

**Reply by thaibm92** [reply]
I just leave it as default when the build is finished

**Reply by arichornlover** [reply]
> 1. Once downloaded, the interface does not automatically disappear

if you don’t know what happened, I was trying to replace the old downloading menu with the one based off of YTMusicUltimate

**Reply by thaibm92** [reply]
Yes, YTMusicUltimate is doing great

**Reply by arichornlover** [reply]
I fixed the problem.
try this one https://www.dropbox.com/scl/fi/lj6w7dhiukmtjoxiz1bc5/YouTube.Reborn.v5.0.0.deb?rlkey=hkfu04hokm1xydcwmpkh8lkxg&dl=1

**Reply by thaibm92** [reply]
The download interface is gone, but there is a frozen progress image
![z5308402754851_adf1347aa82acdf7bc1b1c8722951240](https://github.com/arichornlover/uYouEnhanced/assets/68208369/7d1bd47c-11a0-4be5-8f8e-43f9bd7039e7)

**Reply by thaibm92** [reply]
2. The application will exit when pulled to the left in the toolbar ( download, like, share ....) , this error you can fix?

**Reply by arichornlover** [reply]
Can’t fix the “pulled to the left in the toolbar” error just yet, I haven’t tested the new changes from the deb file in the past week.

**Reply by arichornlover** [reply]
https://www.dropbox.com/scl/fi/p7kmif3y70cfdc45rr88o/YouTube.Reborn.v5.0.0.deb?rlkey=9oiwoanjdg64302mmp31one7c&dl=1
I undid all of the changes to that download menu for now.
also the “YouTube Premium Logo” is fixed in the newest changes aswell.

**Reply by thaibm92** [reply]
- Error "pulled to the left in the toolbar" does not exit the application when "Hide OP button player overlay video" is enabled.
- IMPORT OPTIONS & EXPORT OPTIONS do not work and cause the application to exit.
Are you developing it?
![image](https://github.com/arichornlover/uYouEnhanced/assets/68208369/3309101d-b4ef-477e-b66d-1bcd537a7960)

**Reply by arichornlover** [reply]
> Error "pulled to the left in the toolbar" does not exit the application when "Hide OP button player overlay video" is enabled.

Hmm. Seems like I didn’t fully hide the OP button correctly. That’s something I’m gonna be looking at later.
> IMPORT OPTIONS & EXPORT OPTIONS do not work and cause the application to exit.
Are you developing it?

those import/export buttons don’t have a function. all it will do is just crash the app when you press them.

**Reply by thaibm92** [reply]
Currently, the YouTubeRebornPlus is not working for IPA 19.16.3 do you know it (can't open app), and it also affects reborn v5 , I tried your new release yesterday and it is crashing

**Reply by arichornlover** [reply]
Can you give me the crash log? I gotta see the error message.

also I would assume the crashing happened because in the release of YouTubeRebornPlus v19.16.3-4.2.6
I revamped the `buildapp.yml` & `Makefile` files to be more like uYouEnhanced.

**Reply by thaibm92** [reply]
still cannot be fixed, I don't know what's wrong with it from build to installation, but it doesn't give an error message, only when the installation is finished and it doesn't open like the video below.

https://github.com/arichornlover/uYouEnhanced/assets/68208369/b8c5078c-7ba4-4dec-8613-4a37ea268b8c

**Reply by arichornlover** [reply]
Hey, try rebuilding the package? (ipa) I modified the buildapp.yml file because it wasn’t implementing Reborn correctly so maybe that’s the cause?

or possibly, iOS 17.5 Beta causes it to crash?

**Reply by thaibm92** [reply]
Still can't open the app, I'm using iOS 17.4.1. I also tried installing uYouEnhanced it use good

**Reply by arichornlover** [reply]
Hey @thaibm92 download it again, I found out that it was because of Alderis being outdated 😅

**Reply by galerious** [reply]
Can you tell me when the playback speed control will be available again on uyouenhanced and youtubereborn?

It's a pity that this was a very cool feature😢

**Reply by thaibm92** [reply]
> Hey @thaibm92 download it again, I found out that it was because of Alderis being outdated 😅

It worked but crashed when I logged into my Google account for the first time and the Reborn button didn't show up. Can you update to the main-night branch? I think it's better to use v5.

**Reply by arichornlover** [reply]
Seems like I didn’t include the localization from YouTube Reborn v4/v5, that is my bad.

**Reply by bhackel** [reply]
> Can you tell me when the playback speed control will be available again on uyouenhanced and youtubereborn?
> 
> It's a pity that this was a very cool feature😢

Playback speed control is still in uYouEnhanced - It was made default disabled due to crashing. You can turn it on in uYou -> uYou Settings -> Video Player -> Playback Speed Controls
